### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by ruflin**
Add more integration test to check if filebeat works as expected.

### CODE REVIEW

File: tests/integration/test_crawler.py [hunk 1]
```diff
@@ -3,11 +3,13 @@
 import os
 import time
 
-
+# Additional tests to be added:
+# * Check what happens when file renamed -> no recrawling should happen
```
Comment (andrewkroh) [inline tests/integration/test_crawler.py]: Is that not what `test_file_renaming` does?

File: tests/integration/test_crawler.py [hunk 2]
```diff
@@ -83,3 +83,55 @@ def test_unfinished_line(self):
         assert iterations == i
 
 
+    def test_file_renaming(self):
+        # Makes sure that when a file is renamed, the content is not read again.
+        # Checks that if a line does not have a line ending, is is not read yet
```
Comment (andrewkroh) [inline tests/integration/test_crawler.py]: Copy/paste? I think this command can be removed.

**Comment by monicasarbu** [comment]
Looks good to me

**Comment by andrewkroh** [comment]
Aside from the two minor comments on comments, LGTM.

**Comment by ruflin** [comment]
@andrewkroh I will clean up the comments.

**Pull request merged**